vortex-datafusion/src/persistent/execution.rs

```diff
@@ -151,20 +151,36 @@ impl ExecutionPlan for VortexExec {
     }
 }
 
-fn repartition_by_count(
+fn repartition_by_size(
```

Comment:
FLUP, but let's try to repartition by the uncompressed size of the projected columns, rather than file size

Reply:
That stat isn't exposed by DF right now, but I'm looking into adding it - https://github.com/apache/datafusion/issues/7548